Dockerfile.win10.min

```diff
@@ -134,8 +134,8 @@ LABEL CUDA_VERSION="${CUDA_VERSION}"
 #
 # Installing Tensorrt
 #
-ARG TENSORRT_VERSION=8.5.3.1
-ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-11.8.zip"
+ARG TENSORRT_VERSION=8.6.1.6
+ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-12.0.zip"
```

Comment:
Just double checking that this shouldn't be 12.1?

Reply:
I double checked that value from our download page: 
https://developer.nvidia.com/nvidia-tensorrt-8x-download